example-playbooks/ideal_token_ring/README.md

```diff
@@ -2,6 +2,11 @@
 
 This is an ansible playbook that lets you assign ideal token ring to a known size cluster
 
+## Restrictions and setting expectations
+
+The cluster after using ideal token range HAS to use NetworkTopologyStrategy.
```

Comment:
This is an incorrect statement.
If you generate an ideal token ring using number of nodes + tokens (and not using the inventory file) this is going to generate a ring ideal for a SimpleStrategy.